### ISSUE TITLE
Calling Regexes during FrameCode creation time can corrupt the regex call we are interpreting

### ISSUE THREAD

**Issue by KristofferC** [JuliaDebug/JuliaInterpreter.jl#255]
Adding (on Windows)

```diff
--- a/src/types.jl
+++ b/src/types.jl
@@ -80,6 +80,7 @@ function FrameCode(scope, src::CodeInfo; generator=false, optimize=true)
             src.code[i] = nothing
         end
     end
+    splitdrive("a")
     used = find_used(src)
     return FrameCode(scope, src, methodtables, breakpoints, used, generator)
 end
```

to the `FrameCode` constructor makes the tests fail:

```
Main tests: Test Failed at C:\Users\Kristoffer\Debugging\JuliaInterpreter\test\interpret.jl:97
  Expression: #= C:\Users\Kristoffer\Debugging\JuliaInterpreter\test\interpret.jl:97 =# @interpret(joinpath("/home/julia/base", "sysimg.jl")) == "/home/julia/base/sysimg.jl"
   Evaluated: "/sysimg.jl" == "/home/julia/base/sysimg.jl"
Stacktrace:
 [1] top-level scope at C:\Users\Kristoffer\Debugging\JuliaInterpreter\src\construct.jl:675
 [2] include at .\boot.jl:326 [inlined]
```

On windows `splitdrive` ends up calling `match` on a regex so I guess what is happening is that the call to the regex during creating the `FrameCode` sets some state in PCRE which corrupts the state that is needed for the regex we are currently interpreter. Presumably we need to add some stuff to `compiled_methods`.

**Comment by KristofferC** [comment]
Fixed in https://github.com/JuliaDebug/JuliaInterpreter.jl/pull/256 by adding the corresponding regex calls to `compiled_methods`.

**Comment by KristofferC** [comment]
Closed by #259
